src/cart/crt_iv.c

```diff
@@ -3426,11 +3427,11 @@ crt_iv_update(crt_iv_namespace_t ivns, uint32_t class_id,
 	* All other checks are performed inside of crt_iv_update_interna.
 	*/
 	if (iv_value == NULL) {
-		D_ERROR("iv_value is NULL\n");
-
 		rc = -DER_INVAL;
-		update_comp_cb(ivns, class_id, iv_key, NULL, iv_value,
-			       rc, cb_arg);
+
+		D_ERROR("iv_value is NULL " DF_RC "\n", DP_RC(rc));
```

Comment:
probably should be either "iv_value is NULL; rc: " DF_RC  or just drop printing of rc

Reply:
> "iv_value is NULL; rc: "

Why though? That wouldn't format the rc like DF_RC does.

> just drop printing of rc

I think it's better to print the rc for debugging

Reply:
Including the rc if available is in our coding standard.